myhpom/MyHPOM
# [MH-226] Styles maintenance page for 'coming soon'.

To test this locally you can:

Add `export USE_NGINX=true` to your .env file, and rerun `./hsctl rebuild` -- this will launch an nginx container that is configured to serve via localhost:80

Then you can use the `./hsctl maint_on` and `./hsctl maint_off` commands to turn on and off this page.

I've set up the deploy jenkins jobs to have an additional option one can check to turn on the maintenance screen on deploy: https://ci.mindmyhealth.org/job/Deploy_MMH/
